packages/@sanity/field/src/diff/components/ChangeList.tsx

```diff
@@ -77,61 +82,56 @@ export function ChangeList({diff, fields, schemaType}: Props): React.ReactElemen
   const showFooter = isRoot && changes.length > 1
 
   return (
-    <div
-      className={styles.root}
-      data-revert-all-changes-hover={confirmRevertAllHover ? '' : undefined}
-    >
-      <div className={styles.changeList}>
-        {changes.map((change) => (
-          <ChangeResolver change={change} key={change.key} />
-        ))}
-      </div>
-
-      {showFooter && (
-        <div className={styles.footer}>
-          {isComparingCurrent && (
-            <div className={styles.revertAllContainer} ref={setRevertAllContainerElement}>
-              <Button
-                color="danger"
-                icon={UndoIcon}
-                kind="secondary"
-                onClick={handleRevertAllChangesClick}
-                onMouseEnter={handleRevertAllChangesMouseEnter}
-                onMouseLeave={handleRevertAllChangesMouseLeave}
-                // selected={confirmRevertAllOpen}
-              >
-                Revert all changes
-              </Button>
-            </div>
-          )}
-        </div>
-      )}
-
-      {confirmRevertAllOpen && (
-        <LegacyLayerProvider zOffset="paneFooter">
-          <PopoverDialog
-            actions={[
-              {
-                color: 'danger',
-                action: revertAllChanges,
-                title: 'Revert all',
-              },
-              {
-                kind: 'simple',
-                action: closeRevertAllChangesConfirmDialog,
-                title: 'Cancel',
-              },
-            ]}
-            onAction={handleConfirmDialogAction}
-            onClickOutside={closeRevertAllChangesConfirmDialog}
-            referenceElement={revertAllContainerElement}
-            size="small"
-          >
-            Are you sure you want to revert all {changes.length} changes?
-          </PopoverDialog>
-        </LegacyLayerProvider>
-      )}
-    </div>
+    <Card>
+      <Stack space={5} data-revert-all-changes-hover={confirmRevertAllHover ? '' : undefined}>
+        <Stack as={ChangeListWrapper} space={5}>
+          {changes.map((change) => (
+            <ChangeResolver change={change} key={change.key} />
+          ))}
+        </Stack>
+
+        {showFooter && isComparingCurrent && (
+          <div ref={setRevertAllContainerElement}>
+            <Button
+              tone="critical"
+              mode="ghost"
+              text="Revert all changes"
+              icon={UndoIcon}
+              onClick={handleRevertAllChangesClick}
+              onMouseEnter={handleRevertAllChangesMouseEnter}
+              onMouseLeave={handleRevertAllChangesMouseLeave}
+              // selected={confirmRevertAllOpen}
+              style={{width: '100%'}}
```

Comment:
Nitpick: This inline style can be removed by wrapping the `Button` in a `Stack`.